All EIPIP Meetings/Meeting 012.md

```diff
@@ -0,0 +1,364 @@
+# 1. EIP Improvement Process Call #12 Notes
+### 1.0.1. Meeting Date/Time: Wednesday, July 15, 2020 at 15:00 UTC
+### 1.0.2. Meeting Duration: 1 hr
+### 1.0.3. [Audio/Video of the meeting](https://www.youtube.com/watch?v=5PSQDT1nr2E)
+### 1.0.4. [Agenda](https://github.com/ethereum-cat-herders/EIPIP/issues/23)
+### 1.0.5. Moderator: Pooja Ranjan
+### 1.0.6. Notes: Alita Moore
+
+## 1.1. Decisions Made
+
+Decision Item | Description
+---|---
+**12.1** | Edson will make a survey to the current editors on the expectations from an editor (time-commitment, qualifications)
+**12.2** | James wil look into the necessary permissions and how to setup users with triaging capabilities
+**12.3** | A process for removing editors and criteria for removing editors is a future item to discuss
+**12.4** | Link to the live EFI document through EIP 1 rather than continuously updating EIP 1; EIP 1 can be used as a hub for statuses instead.
+**12.5** | Remove emergency EIP process item until such time that it becomes pertinent
+
+
+# 2. [Agenda](https://github.com/ethereum-cat-herders/EIPIP/issues/23#issue-655857894)
+
+
+1. Onboarding EIP editors, new EIP validator and suggestions.
+2. Separation of EIP process and hard-fork coordination
+3. Discuss the EIP-1 Brainstorming Document.
+    3.a Goals Outcomes
+    3.b What we are not doing.
+    3.c EIPs TODO
+4. Clarifications for how a precompile address is decided.
+5. Emergency communication document and checklist
+6. EIPIP project board.
+7. Postmortem format feedback
+8. Review action items from previous meeting
+
+---
+
+# 3. Onboarding EIP editors, new EIP validator and suggestions

```

Comment:
State the number as per Agenda item number. eg.
# 1. Onboarding EIP editors, new EIP validator and suggestions